.github/workflows/deploy.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
     services:
       postgres:
-        image: postgres:9.6
+        image: postgres:latest
```

Comment:
Wouldn't it be better to pin it to a specific version? (the same version as we are using in production?)